cluster-provision/k8s/1.29/extra-pre-pull-images

```diff
@@ -2,10 +2,10 @@ quay.io/kubevirtci/install-cni:1.15.0
 quay.io/kubevirtci/operator:1.15.0
 quay.io/kubevirtci/pilot:1.15.0
 quay.io/kubevirtci/proxyv2:1.15.0
-quay.io/calico/cni:v3.18.0
-quay.io/calico/kube-controllers:v3.18.0
-quay.io/calico/node:v3.18.0
-quay.io/calico/pod2daemon-flexvol:v3.18.0
+quay.io/calico/cni:v3.25.2
+quay.io/calico/kube-controllers:v3.25.2
+quay.io/calico/node:v3.25.2
+quay.io/calico/pod2daemon-flexvol:v3.25.2
```

Comment:
@brianmcarey Do we need to keep "quay.io/calico/pod2daemon-flexvol:v3.25.2" entry in cluster-provision/k8s/1.*/extra-pre-pull-images files? As I see it is no more being referred in the newer cni.do-not-change.yaml files, unlike with v3.18.0 cni.do-not-change.yaml was having a reference like below:

>          - name: flexvol-driver
>            image: docker.io/calico/pod2daemon-flexvol:v3.18.0

Not sure if this image is required else where than cni yaml file. In earlier PR to bump calico to 3.27.2 #1281, I removed the line "quay.io/calico/pod2daemon-flexvol:v3.25.2" from extra-pre-pull-images.

Reply:
Good catch - thanks @chandramerla  - removed.